pkg/controllers/cloud_config_sync_controller_test.go

```diff
@@ -393,6 +394,19 @@ var _ = Describe("Cloud config sync reconciler", func() {
 		Expect(len(allCMs.Items)).To(BeZero())
 	})
 
+	It("should perform config sync for AWS platform if there is a reference in infra resource", func() {
+		infraResource := makeInfrastructureResource(configv1.AWSPlatformType)
+		Expect(cl.Create(ctx, infraResource)).To(Succeed())
+		infraResource.Status = makeInfraStatus(infraResource.Spec.PlatformSpec.Type)
+		Expect(cl.Status().Update(ctx, infraResource.DeepCopy())).To(Succeed())
+		_, err := reconciler.Reconcile(context.TODO(), ctrl.Request{})
+		Expect(err).To(BeNil())
+		allCMs := &corev1.ConfigMapList{}
+		Expect(cl.List(ctx, allCMs, &client.ListOptions{Namespace: targetNamespaceName})).To(Succeed())
+		Expect(len(allCMs.Items)).NotTo(BeZero())
+		Expect(len(allCMs.Items)).To(BeEquivalentTo(1))
```

Comment:
Can we add a little whitespace to make this easier to follow
```suggestion
		infraResource := makeInfrastructureResource(configv1.AWSPlatformType)
		Expect(cl.Create(ctx, infraResource)).To(Succeed())
		
		infraResource.Status = makeInfraStatus(infraResource.Spec.PlatformSpec.Type)
		Expect(cl.Status().Update(ctx, infraResource.DeepCopy())).To(Succeed())
		
		_, err := reconciler.Reconcile(context.TODO(), ctrl.Request{})
		Expect(err).To(BeNil())
		
		allCMs := &corev1.ConfigMapList{}
		Expect(cl.List(ctx, allCMs, &client.ListOptions{Namespace: targetNamespaceName})).To(Succeed())
		
		Expect(len(allCMs.Items)).NotTo(BeZero())
		Expect(len(allCMs.Items)).To(BeEquivalentTo(1))
```